app/assets/stylesheets/styles/components/guides/ui/_actions_modal.css.scss

```diff
@@ -1,13 +1,13 @@
 .actions-modal {
-  position: fixed;
-  font-family: Arial, Helvetica, sans-serif;
-  top: 0;
+  /*position: fixed;*/
```

Comment:
Foundation provides this out of the box in styling, so we don't need to do all of this. I commented it out, but it can probably safely be deleted.